recipes/redis-plus-plus/all/conanfile.py

```diff
@@ -115,10 +115,9 @@ def package(self):
 
     def package_info(self):
         self.cpp_info.set_property("cmake_file_name", "redis++")
-        self.cpp_info.set_property("cmake_target_name", "redis++")
+        self.cpp_info.set_property("cmake_target_name", "redis++::redis++")
         self.cpp_info.set_property("pkg_config_name", "redis++")
-        self.cpp_info.components["redis++lib"].set_property("cmake_find_package", "redis++" + "_static" if not self.options.shared else "")
-        self.cpp_info.components["redis++lib"].set_property("cmake_target_name", "redis++" + "_static" if not self.options.shared else "")
+        self.cpp_info.components["redis++lib"].set_property("cmake_target_name", "redis++::redis++" + "_static" if not self.options.shared else "")
```

Comment:
```suggestion
```

Reply:
Or to keep until conan v2 to avoid to create 2 targets? I don't know if CMakeDeps handles this case.

Reply:
It's not needed. I have already checked for both modern and legacy generators and its working